seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/sink/KafkaSinkFactory.java

```diff
@@ -39,17 +36,9 @@ public String factoryIdentifier() {
     @Override
     public OptionRule optionRule() {
         return OptionRule.builder()
-                .required(Config.FORMAT, Config.BOOTSTRAP_SERVERS)
-                .conditional(
```

Comment:
Why remove this? `conditional` option can help SeaTunnel Web auto create connector form.

Reply:
because i find this rule is only check `topic` parameter exist when format is in  (json, cancal_json, text, ....), but it required whether the format is. so i remove it.
if the web need use this, i will add this `conditional` option back.

Reply:
> Why remove this? `conditional` option can help SeaTunnel Web auto create connector form.

@Hisoka-X i remove it before, but eric said it will be used in web. so i put it back and move the topic option check in conditional, just put it to required